aggregations/db_tables/daily_add_key_by_app_count.py

```diff
@@ -0,0 +1,94 @@
+import datetime
+
+from . import DAY_LEN_SECONDS, daily_start_of_range, time_json
+from ..periodic_aggregations import PeriodicAggregations
+
+
+class DailyAddKeyByAppCount(PeriodicAggregations):
+    def dependencies(self) -> list:
+        return ["near_ecosystem_entities"]
+
+    @property
+    def sql_create_table(self):
+        return """
+            CREATE TABLE IF NOT EXISTS daily_add_key_by_app_count
+            (
+                collected_for_day  DATE NOT NULL,
+                app_id TEXT NOT NULL,
+                receipt_receiver_account_id TEXT NOT NULL,
+                add_key_count INTEGER NOT NULL,
+                PRIMARY KEY (collected_for_day, app_id, receipt_receiver_account_id )
+            );
+            CREATE INDEX IF NOT EXISTS daily_add_key_by_app_count_idx
+                ON daily_add_key_by_app_count (collected_for_day, add_key_count DESC)
+        """
+
+    @property
+    def sql_drop_table(self):
+        return """
+            DROP TABLE IF EXISTS daily_add_key_by_app_count
+        """
+
+    @property
+    def sql_select(self):
+        all_apps = """
+            SELECT token as contract_id, 
+            slug as app_id
+            FROM public.near_ecosystem_entities e, unnest(string_to_array(e.contract, ', ')) s(token)
+            WHERE app = TRUE AND length(contract)>0
+        """
+
+        with self.analytics_connection.cursor() as analytics_cursor:
+            analytics_cursor.execute(all_apps)
+            app_contracts = analytics_cursor.fetchall()
```

Comment:
While Python allows creating variable in a block of code and using it after we quit this block, I feel it's not the best practice and it makes the reading more complicated. 
I suggest you to move this block to a function with `app_contracts` as the returned value.

Reply:
I'll do you one even better...  This whole thing it overly complicated &  SQL injection is terrible for analytics.  I did not want to built it this way.

What do you think about pushing the call to  table near_ecosystem_entities to the other job 'daily_new_accounts_by_app_count'?